scripts/ping360_socat.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python -u
+
+"""
+This manage socat sessions so that every Ping360 device has a screen session
+running socat to expose it to an UDP port.
+"""
+import time
+import subprocess
+import re
+
+def list_ping_devices():
+    """
+    List devices in dev/serial/ping/ping360*
+    """
+    try:
+        # Note: we could easily support Ping1D here.
+        output = subprocess.check_output("ls /dev/serial/ping/Ping360*", shell=True)
+        output = [line for line in output.decode().split("\n") if len(line) > 0]
+        return output
+    except subprocess.CalledProcessError as e:
+        print("Error listing devices at '/dev/serial/ping/ping360*':", e)
+        return []
+
+def device_has_screen(device):
+    """
+    Checks if 'device' has an associated screen session
+    returns true if so
+    """
+    screen_name = device.split("/")[-1].split("-t-")[0]
```

Comment:
Can you comment what is going on here ? What is "-t-" and what is the device format ?

Reply:
The idea is to shorten the device name so we don't have a huge name for the screen name. `Ping360-id-1` instead of `Ping360-id-1-r-67-v-3.0.1`, for instance.
On the other hand, maybe leaving the full name is better, this way all the information is available at the companion System page